### DISCUSSION TITLE
Is BPF multiple condition example wrong?

### DISCUSSION THREAD

**Discussion by Ron89** [Security-Onion-Solutions/securityonion#10116] [Unsupported Versions] [answered]
I am referring to https://docs.securityonion.net/en/2.3/bpf.html#multiple-conditions.

In the quoting (https://docs.securityonion.net/en/2.3/bpf.html#quoting) part of the document, `&` and `|` are already noted as reserved symbol and need to be quoted. But in this `multiple-condition` section, the example didn't quote the condition linking symbols `||` and `&&`. Is it a mistake or intended?

**Comment by dougburks** [comment, marked as answer]
I've updated https://docs.securityonion.net/en/2.3/bpf.html#multiple-conditions to simplify it.
